ci-operator/jobs/openshift/release/openshift-release-release-4.5-periodics.yaml

```diff
@@ -6077,3 +6077,75 @@ periodics:
     - name: release-pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cluster: api.ci
+  decorate: true
+  interval: 48h
+  labels:
+    ci.openshift.io/release-type: informing
+    job-release: "4.5"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: release-openshift-ocp-installer-e2e-metal-ipi-4.5
```

Comment:
So this job does look a little different than for example `release-openshift-ocp-installer-e2e-azure-4.5`. That one has some env vars to specify RELEASE_IMAGE_LATEST. It also has a configMap called prow-job-cluster-launch-installer-e2e.

And in other jobs we've enabled this, we've had to map e2e-metal-ipi to the workflow in the step registry, where does that happen for this job?

Reply:
This is what I've asked in the forum: all the jobs here are template based, while our is multi-stage based, so it's slightly different, and it was not clear the level of integration required.

Reply:
You need the template parameters, and if you aren't respecting them that would be an issue (it means we can't run upgrade jobs).

In general as the first adopter of steps there may be under documented steps, but we can find them out as we go.  You definitely need the BRANCH, RELEASE_IMAGE_LATEST, and RELEASE_IMAGE_INITIAL optional parameters.

Reply:
We don't use RELEASE_IMAGE_INITIAL in the multi-stage test, maybe we can omit it. Is it ok to use `registry.svc.ci.openshift.org/ocp/release:4.5-ci`?